src/fmopl.c

```diff
@@ -657,7 +657,7 @@ static int OPLOpenTable( void )
 		TL_TABLE[t] = TL_TABLE[TL_MAX+t] = 0;
 	}
 
-	/* make sinwave table (total level offet) */
+	/* make sinwave table (total level offset) */
```

Comment:
Just noticed this should really be "sine wave" instead of "sinwave" too

Reply:
Added requested change in 07c832c